tests/e2e/test_splunk_addon.py

```diff
@@ -137,7 +162,9 @@ def test_splunk_connection_docker(testdir):
 
     # run pytest with the following cmd args
     result = testdir.runpytest(
-        "--splunk-type=docker",
+        "--splunk-type=kubernetes",
+        "--splunk-web-scheme=http",
+        "--splunk-version={0}".format(request.config.getoption("splunk_version")),
```

Comment:
Does this change mean that all our old tests were not taking into account a Splunk version passed?

https://github.com/splunk/pytest-splunk-addon/blob/adc01f7ae6ad93f1607e13f236511eff1df9c075/.github/workflows/build-test-release.yml#L231

Reply:
Yes, old tests were not taking a passed Splunk Version to execute the tests upon, they were by default taking the `latest` splunk-version only.